apis/kueue/v1beta1/workload_types.go

```diff
@@ -124,6 +124,9 @@ type PodSet struct {
 	//
 	// +optional
 	MinCount *int32 `json:"minCount,omitempty"`
+
+	// Reference on the Pod template.
+	PodTemplateRef *corev1.ObjectReference `json:"podTemplateRef,omitempty"`
```

Comment:
An ObjectReference might be an overkill for this. Because the PodTemplate object should be in the same namespace as the Workload, we just need one string: the name.

Although we usually create a type alias for this. See for example the QueueName field in WorkloadSpec

Reply:
Probably, it's better to rename this field to PodTemplateName?

Reply:
sounds good